freeze/2.7.txt

```diff
@@ -67,7 +67,7 @@ funcsigs==1.0.2
 future==0.17.1
 futures==3.3.0
 google-auth==1.6.3
-hcloud==1.4.0
+hcloud==1.4.1
```

Comment:
The hcloud changes should be done in a separate PR. Also, instead of manually updating the freeze files, go through the `make update && make freeze` process. I recommend doing so after the changes in this PR are merged.